CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ Versioning](https://semver.org/spec/v2.0.0.html) for `libnopegl`.
 - `ngl_scene_get_filepaths()` and `ngl_scene_update_filepath()` to introspect
   and manipulate the filepath based parameters in the scene graph
 - Live change support for `Media.filename`
+- `ngl_get_viewport()` to get the viewport currently in use in the rendering
```

Comment:
in -> by ?

Reply:
Fixed (header doxy was using "by" so no update required there)